requirements.txt

```diff
@@ -1,5 +1,5 @@
 mako
-docker-py==1.5.0
+git+https://github.com/rancherio/docker-py.git@1.5.0-rancher
```

Comment:
Had to patch because of a simple bug where docker-py was calling /volumes instead of /volumes/create.
They have had it fixed on master for quite some time but have not done a release in awhile.